evaluation_plans/osps/build_release/evaluations.go

```diff
@@ -93,7 +93,6 @@ func OSPS_BR_03() (evaluation *layer4.ControlEvaluation) {
 			"Maturity Level 3",
 		},
 		[]layer4.AssessmentStep{
-			ensureGitHubWebsiteLinkUsesHTTPS,
```

Comment:
This was removed because I can't find a reference to `websiteUrl` in the REST API docs for either the GetRepository or GetOrganization operations.

Reply:
Yep, something went sideways there